agent/consul/internal_endpoint.go

```diff
@@ -43,7 +43,14 @@ func (m *Internal) NodeInfo(args *structs.NodeSpecificRequest,
 				return err
 			}
 
-			reply.Index, reply.Dump = index, dump
+			// old UIs may not handle .Dump so we added all peered nodes in .ImportedDump
```

Comment:
also, from chatting with Freddy, we are changing this to use `reply.Dump` in either case.

It makes sense bc old UIs, for instance, won't ask for peer nodes